app/models/verification_type.rb

```diff
@@ -32,7 +32,7 @@ class VerificationType
   OUTSTANDING_STATES = %w[outstanding rejected].freeze
   DUE_DATE_STATES = %w[review outstanding rejected].freeze
   SATISFIED_STATES = %w[verified attested valid curam].freeze
-  NEGATIVE_STATES = %w[negative_response_received].freeze
+  NEGATIVE_STATES = %w[outstanding review rejected unverified pending].freeze
```

Comment:
From what I could find, this constant was not being used anywhere, so I figured we could repurpose this for use in the scripts.

Reply:
Interesting that it's not used anywhere.  I'm wary of repurposing it though because I think the meaning of "negative" may be more specific than all of these states.  We can check on this.